content/en/account_management/rbac/permissions.md

```diff
@@ -21,27 +21,17 @@ After creating a role, assign or remove permissions to this role directly by [up
 
 ## Overview
 
-### General permissions
```

Comment:
Are there still users with access to these (now deprecated) permissions? That would be my only concern. Otherwise, I'm fine with removing this section, assuming we don't have some type of deprecation policy where we have to leave notice of it in the docs or elsewhere for a certain period of time.

Reply:
As of now, there still are, but our end goal (which should be accomplished soon) is to remove `standard` from the UI entirely. `admin` already has been removed from the UI.

They both still exist in our database, but from a user's perspective, the `admin` permission does not exist, and `standard` will be the same way soon.